test/JDBC/input/case_insensitive_collation-vu-prepare.sql

```diff
@@ -0,0 +1,38 @@
+-- BABEL-3094
+-- sys table and view columns need to have the correct output type and collation
```

Comment:
we should have all these test cases in upgrade tests. Prepare objects in 13.x and we should check this case insensitive verification in 14.x

Reply:
done